Source/WebCore/page/PointerLockController.cpp

```diff
@@ -155,6 +207,8 @@ void PointerLockController::didAcquirePointerLock()
 void PointerLockController::didNotAcquirePointerLock()
 {
     enqueueEvent(eventNames().pointerlockerrorEvent, m_element.get());
+    // Typically this is because the window does not have focus, hence WrongDocumentError.
+    rejectPromises(WrongDocumentError);
```

Comment:
Is there anything in the spec about this?

Reply:
Yes, this is the relevant section: https://github.com/w3c/pointerlock/blob/cf54ad840997a702583b828f43b216ae0496ad20/index.html#L338

I can expand the comment to include the text for the spec to make it more clear.

Reply:
Done.